From 9aa69000d00a17ab72012c33452b631178812965 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:35:54 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(8e09e639b)..lineage-16.0-android-9.0.0_r31(3fb8017c7) (DO
 NOT SUBMIT)

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

Miao Wang (1):
 - Update SDK atree to include librsjni_androidx

Raju Kulkarni (1):
 - Build tools Version number bump for release with Studio 3.2 This is a
   SDK change only and does not impact real devices Bug: 115528420 Test:
   N/A

android-build-team Robot (1):
 - Snap for 5044688 from b0a428fd03f31d0d118821b0462fc2925eb6000c to
   pi-qpr2-release

Change-Id: I4428d8e95e9f47a4643530a7895d95124de701e8

Change-Id: I1b436b72cd2342dc74cf06d2a3ac26392a4bea46
---
 build/sdk.atree                      | 48 +++++++++++++++-------------
 sdk/build_tools_source.prop_template |  2 +-
 2 files changed, 27 insertions(+), 23 deletions(-)

diff --git a/build/sdk.atree b/build/sdk.atree
index acfc8c553..382e46213 100644
--- a/build/sdk.atree
+++ b/build/sdk.atree
@@ -94,37 +94,41 @@ external/clang/LICENSE.TXT                    build-tools/${PLATFORM_NAME}/rende
 prebuilts/sdk/renderscript/lib/javalib.jar            build-tools/${PLATFORM_NAME}/renderscript/lib/androidx-rs.jar
 prebuilts/sdk/renderscript/lib/javalib_legacy.jar     build-tools/${PLATFORM_NAME}/renderscript/lib/renderscript-v8.jar
 
-prebuilts/sdk/renderscript/lib/arm/libc.so            build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/armeabi-v7a/libc.so
-prebuilts/sdk/renderscript/lib/arm/libm.so            build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/armeabi-v7a/libm.so
-prebuilts/sdk/renderscript/lib/arm/libcompiler_rt.a   build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/armeabi-v7a/libcompiler_rt.a
-prebuilts/sdk/renderscript/lib/arm/libRSSupport.so    build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/armeabi-v7a/libRSSupport.so
-prebuilts/sdk/renderscript/lib/arm/librsjni.so        build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/armeabi-v7a/librsjni.so
-prebuilts/sdk/renderscript/lib/arm/libblasV8.so       build-tools/${PLATFORM_NAME}/renderscript/lib/blas/armeabi-v7a/libblasV8.so
-prebuilts/sdk/renderscript/lib/arm/librsrt_arm.bc     build-tools/${PLATFORM_NAME}/renderscript/lib/bc/armeabi-v7a/libclcore.bc
-
-prebuilts/sdk/renderscript/lib/x86/libc.so            build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/x86/libc.so
-prebuilts/sdk/renderscript/lib/x86/libm.so            build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/x86/libm.so
-prebuilts/sdk/renderscript/lib/x86/libcompiler_rt.a   build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/x86/libcompiler_rt.a
-prebuilts/sdk/renderscript/lib/x86/libRSSupport.so    build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/x86/libRSSupport.so
-prebuilts/sdk/renderscript/lib/x86/librsjni.so        build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/x86/librsjni.so
-prebuilts/sdk/renderscript/lib/x86/libblasV8.so       build-tools/${PLATFORM_NAME}/renderscript/lib/blas/x86/libblasV8.so
-prebuilts/sdk/renderscript/lib/x86/librsrt_x86.bc     build-tools/${PLATFORM_NAME}/renderscript/lib/bc/x86/libclcore.bc
+prebuilts/sdk/renderscript/lib/arm/libc.so               build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/armeabi-v7a/libc.so
+prebuilts/sdk/renderscript/lib/arm/libm.so               build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/armeabi-v7a/libm.so
+prebuilts/sdk/renderscript/lib/arm/libcompiler_rt.a      build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/armeabi-v7a/libcompiler_rt.a
+prebuilts/sdk/renderscript/lib/arm/libRSSupport.so       build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/armeabi-v7a/libRSSupport.so
+prebuilts/sdk/renderscript/lib/arm/librsjni.so           build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/armeabi-v7a/librsjni.so
+prebuilts/sdk/renderscript/lib/arm/librsjni_androidx.so  build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/armeabi-v7a/librsjni_androidx.so
+prebuilts/sdk/renderscript/lib/arm/libblasV8.so          build-tools/${PLATFORM_NAME}/renderscript/lib/blas/armeabi-v7a/libblasV8.so
+prebuilts/sdk/renderscript/lib/arm/librsrt_arm.bc        build-tools/${PLATFORM_NAME}/renderscript/lib/bc/armeabi-v7a/libclcore.bc
+
+prebuilts/sdk/renderscript/lib/x86/libc.so               build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/x86/libc.so
+prebuilts/sdk/renderscript/lib/x86/libm.so               build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/x86/libm.so
+prebuilts/sdk/renderscript/lib/x86/libcompiler_rt.a      build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/x86/libcompiler_rt.a
+prebuilts/sdk/renderscript/lib/x86/libRSSupport.so       build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/x86/libRSSupport.so
+prebuilts/sdk/renderscript/lib/x86/librsjni.so           build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/x86/librsjni.so
+prebuilts/sdk/renderscript/lib/x86/librsjni_androidx.so  build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/x86/librsjni_androidx.so
+prebuilts/sdk/renderscript/lib/x86/libblasV8.so          build-tools/${PLATFORM_NAME}/renderscript/lib/blas/x86/libblasV8.so
+prebuilts/sdk/renderscript/lib/x86/librsrt_x86.bc        build-tools/${PLATFORM_NAME}/renderscript/lib/bc/x86/libclcore.bc
 
 prebuilts/sdk/renderscript/lib/arm64/libc.so              build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/arm64-v8a/libc.so
 prebuilts/sdk/renderscript/lib/arm64/libm.so              build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/arm64-v8a/libm.so
 prebuilts/sdk/renderscript/lib/arm64/libcompiler_rt.a     build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/arm64-v8a/libcompiler_rt.a
 prebuilts/sdk/renderscript/lib/arm64/libRSSupport.so      build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/arm64-v8a/libRSSupport.so
 prebuilts/sdk/renderscript/lib/arm64/librsjni.so          build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/arm64-v8a/librsjni.so
+prebuilts/sdk/renderscript/lib/arm64/librsjni_androidx.so build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/arm64-v8a/librsjni_androidx.so
 prebuilts/sdk/renderscript/lib/arm64/libblasV8.so         build-tools/${PLATFORM_NAME}/renderscript/lib/blas/arm64-v8a/libblasV8.so
 prebuilts/sdk/renderscript/lib/arm64/librsrt_arm64.bc     build-tools/${PLATFORM_NAME}/renderscript/lib/bc/arm64-v8a/libclcore.bc
 
-prebuilts/sdk/renderscript/lib/x86_64/libc.so             build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/x86_64/libc.so
-prebuilts/sdk/renderscript/lib/x86_64/libm.so             build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/x86_64/libm.so
-prebuilts/sdk/renderscript/lib/x86_64/libcompiler_rt.a    build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/x86_64/libcompiler_rt.a
-prebuilts/sdk/renderscript/lib/x86_64/libRSSupport.so     build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/x86_64/libRSSupport.so
-prebuilts/sdk/renderscript/lib/x86_64/librsjni.so         build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/x86_64/librsjni.so
-prebuilts/sdk/renderscript/lib/x86_64/libblasV8.so        build-tools/${PLATFORM_NAME}/renderscript/lib/blas/x86_64/libblasV8.so
-prebuilts/sdk/renderscript/lib/x86_64/librsrt_x86_64.bc   build-tools/${PLATFORM_NAME}/renderscript/lib/bc/x86_64/libclcore.bc
+prebuilts/sdk/renderscript/lib/x86_64/libc.so               build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/x86_64/libc.so
+prebuilts/sdk/renderscript/lib/x86_64/libm.so               build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/x86_64/libm.so
+prebuilts/sdk/renderscript/lib/x86_64/libcompiler_rt.a      build-tools/${PLATFORM_NAME}/renderscript/lib/intermediates/x86_64/libcompiler_rt.a
+prebuilts/sdk/renderscript/lib/x86_64/libRSSupport.so       build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/x86_64/libRSSupport.so
+prebuilts/sdk/renderscript/lib/x86_64/librsjni.so           build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/x86_64/librsjni.so
+prebuilts/sdk/renderscript/lib/x86_64/librsjni_androidx.so  build-tools/${PLATFORM_NAME}/renderscript/lib/packaged/x86_64/librsjni_androidx.so
+prebuilts/sdk/renderscript/lib/x86_64/libblasV8.so          build-tools/${PLATFORM_NAME}/renderscript/lib/blas/x86_64/libblasV8.so
+prebuilts/sdk/renderscript/lib/x86_64/librsrt_x86_64.bc     build-tools/${PLATFORM_NAME}/renderscript/lib/bc/x86_64/libclcore.bc
 
 # apksigner
 bin/apksigner                                 build-tools/${PLATFORM_NAME}/apksigner
diff --git a/sdk/build_tools_source.prop_template b/sdk/build_tools_source.prop_template
index f0c887bbe..d89d1027d 100644
--- a/sdk/build_tools_source.prop_template
+++ b/sdk/build_tools_source.prop_template
@@ -1,3 +1,3 @@
 Pkg.UserSrc=false
-Pkg.Revision=${PLATFORM_SDK_VERSION}.0.2
+Pkg.Revision=${PLATFORM_SDK_VERSION}.0.3
 #Pkg.Revision=28.0.0
-- 
2.17.1

